plugins/hypervisors/baremetal/src/main/java/org/apache/cloudstack/api/DeleteBaremetalRctCmd.java

```diff
@@ -64,7 +64,7 @@ public void execute() throws ResourceUnavailableException, InsufficientCapacityE
             SuccessResponse response = new SuccessResponse(getCommandName());
             setResponseObject(response);
         } catch (Exception e) {
-            s_logger.warn(String.format("unable to add baremetal RCT[%s]", getId()), e);
+            s_logger.warn(String.format("unable to delete baremetal RCT[%s]", getId()), e);
             throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, e.getMessage());

```

Comment:
The same thing here regarding the passing of exception to the new one that is thrown.

Reply:
Sure, I'll do one more commit